docs/ml/vision/mlmodel.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "mlmodel"
+title: "Configure an mlmodel Detector or Classifier"
```

Comment:
```suggestion
title: "Configure a Machine Learning Detector or Classifier"
```
[opt] I think these would all read better and be more quickly understandable as normal sentences, similar to how many of the camera models are done.
The first sentence of each of these should include the model name, though: "`mlmodel` is a machine learning detector that draws...."

I realize this is a little beyond scope so feel free to disregard, especially for the segmenters. I realize it's a bit harder to change some of the segmenters into a short sentence since they do very specific things. This one is easily doable but if you think consistency is more important can leave this one as-is too.

Reply:
Sounds good! Made a ticket to investigate later!